test/NuGet.Core.Tests/NuGet.PackageManagement.Test/AuditCheckerTests.cs

```diff
@@ -927,6 +927,62 @@ public void CreateWarnings_WithAuditSettings_RaisesWarningsForProjectsWithMatchi
             packagesWithReportedAdvisories[0].Should().Be(packageIdentity);
         }
 
+        [Theory]
+        [InlineData("true")]
+        [InlineData("false")]
+        public async Task CheckPackageVulnerabilitiesAsync_WithEmptyPackages_ReturnsNoopAuditResult(string enableAudit)
+        {
+            // Arrange
+            var packageSources = new List<SourceRepository>
+            {
+                new SourceRepository(new PackageSource("https://api.nuget.org/v3/index.json"), Repository.Provider.GetCoreV3())
```

Comment:
Can we use the mocks the other tests are using? 

We try not to add nuget.org dependent tests (even if this one is technically testing that we're not calling nuget.org).

Reply:
Good catch. The other tests aren't using true mocks, but they are pointing to a contrived URL. Is that what you mean here?

Reply:
This should be addressed. Please take another look! :-)